src/mlpack/core/data/serialization_shim.hpp

```diff
@@ -278,8 +292,10 @@ template<typename T>
 struct FirstNormalArrayShim
 {
   //! Construct the first shim with the given objects, length, and name.
-  FirstNormalArrayShim(T* t, const size_t len, const std::string& name) :
-      t(t), len(len), name(name) { }
+  FirstNormalArrayShim(T* t, const size_t len, const std::string& name)
+      : t(t), len(len), name(name)
+  {
+  }
```

Comment:
I think it's also fine to have an empty function as `{ }` on the line of the definition.